share/change.py

```diff
@@ -298,7 +298,7 @@ def __init__(self, graph, id, type, attrs, namespace=None):
         self.namespace = namespace
 
         if not self.is_blank:
-            self.instance = IDObfuscator.load(self.id, None)
+            self.instance = IDObfuscator.resolve(self.id)
```

Comment:
I changed `resolve` to use `get_canonical`, so it'll follow the `same_as` chain. I didn't notice that changed its behavior to return None instead of raising NotFoundError... How about I change `load` to use `get_canonical`, and get rid of `resolve`?